VERSIONING.md

```diff
@@ -0,0 +1,19 @@
+In order to support backwards and forwards compatibility, SBP messages
+are versioned and follow a small number of conventions, enabling
+devices to indicate understood message formats, capabilities and
+features. SBP uses a major, minor numbering scheme to indicate
+versions, where major and minor are represented by 8-bit values. The
+minor number is incremented by each message addition, deprecation. The
+major number is incremented when the format of a message is
```

Comment:
Just wanted to make sure that's what it was referring to. 

A mention of stable vs unstable/draft, similar to what we have in the datasheet would be good: "By convention, we refer to messages as `stable` and `draft`. Stable messages are intended for broad use, like `MsgBaselineECEF`, while draft ones are intended for internal development.`.